server/stt/parser.py

```diff
@@ -57,6 +58,32 @@ def parse_inline_content(self, tree, item):
 
         return content
 
+    def _clean_body_html(self, body_elt):
+        """
+        This function retains the core’s cleaner behaviour while forcing <a> tags onto the allow-list
+        for html tags so ingesting doesn't ignore the hyperlinks
+        """
+        if body_elt is None:
+            return body_elt
+
+        allow_tags = list(get_app_config("HTML_TAGS_WHITELIST"))
+        if "a" not in allow_tags:
```

Comment:
I wouldn't do it here, if there is a config we should follow. you can add `a` to `HTML_TAGS_WHITELIST` in `settings.py`